src/components/Menu/Menu.tsx

```diff
@@ -56,24 +68,32 @@ export const Menu = ({ className, ...other }: MenuProps) => {
 
 /**
  * A button that when clicked, shows or hides the Options
+ * Refer to Button documentation for more style variants.
```

Comment:
Could you also add documentation for the new `Menu.Button` functionality to the top-level `Menu` comment so it shows up readily in the Storybook Docs tab?